react/features/blur/middleware.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import { MiddlewareRegistry } from '../base/redux';
+import { getLocalVideoTrack } from '../../features/base/tracks';
+
+import {
+    TOGGLE_BLUREFFECT
+} from './actionTypes';
+
+const logger = require('jitsi-meet-logger').getLogger(__filename);
+
+/**
+ * The redux middleware to handle the video updates in a React way.
```

Comment:
`* The redux middleware to handle the video updates in a React way.`

Isn't it just start/stop the video blur?

Reply:
Updated comment